native/ios/Podfile.lock

```diff
@@ -732,6 +732,6 @@ SPEC CHECKSUMS:
   Yoga: 4bd86afe9883422a7c4028c00e34790f560923d6
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: 9f4f5fc0d4639de209a92fd46798bf1ba48b1f6e
+PODFILE CHECKSUM: 26c04831545c22b7526a71e0f520caedf9e5fafb
```

Comment:
I update this manually, it would be cool if somebody with a mac could run `pod exec pod install` and see whether there are changes.

Reply:
This seems to be fine, no changes after executing pod install :+1: